index.html

```diff
@@ -278,47 +418,665 @@ <h4>Identification and Profile Signaling</h4>
           </thead>
-
           <tbody>
             <tr>
-              <td>DAPT</td>
-              <td><code>dapt</code></td>
+              <td>XML</td>
+              <td><code>xml</code></td>
+              <td><code>http://www.w3.org/XML/1998/namespace</code></td>
+              <td>[[xml-names]]</td>
+            </tr>
+            <tr>
+              <td>TT</td>
+              <td><code>tt</code></td>
+              <td><code>http://www.w3.org/ns/ttml</code></td>
+              <td>[[TTML2]]</td>
+            </tr>
+            <tr>
+              <td>TT Parameter</td>
+              <td><code>ttp</code></td>
+              <td><code>http://www.w3.org/ns/ttml#parameter</code></td>
+              <td>[[TTML2]]</td>
+            </tr>
+            <tr>
+              <td>TT Feature</td>
+              <td><em>none</em></td>
+              <td><code>http://www.w3.org/ns/ttml/feature/</code></td>
+              <td>[[TTML2]]</td>
+            </tr>
+            <tr>
+              <td>TT Audio Style</td>
+              <td><em>tta:</em></td>
+              <td><code>http://www.w3.org/ns/ttml#audio</code></td>
+              <td>[[TTML2]]</td>
+            </tr>
+            <tr>
+              <td>DAPT Metadata</td>
+              <td><code>daptm</code></td>
               <td><code>http://www.w3.org/ns/ttml/profile/dapt#metadata</code></td>
+              <td><em>This specification</em></td>
             </tr>
           </tbody>
-          <p class="issue" data-number="22"></p>
         </table>
-        <p>The namespaces defined by this specification are mutable [[namespaceState]]; all undefined names in these namespaces are reserved for future standardization by the W3C.</p>
-        <p>The namespace prefix values defined above are for convenience and Document Instances MAY use any prefix value that conforms to [[!xml-names]].</p>
-        <p>Other namespaces may be present, as needed and permitted by XML and TTML, use for either specific TTML attributes or elements, or for elements and attributes from other standards or proprietary languages.</p>
-        <p>Implementations shall ignore elements and attributes in the DAPT namespaces that they do not recognize.</p>
-        <p>Implementations shall ignore elements and attributes in non TTML-namespaces that they do not recognize.</p>
-        <p class="issue" data-number="23"></p>
+        <p>
+          The namespace prefix values defined above are for convenience and Document Instances MAY use any prefix value that conforms to [[xml-names]].
+        </p>
+        <p>
+          The namespaces defined by this proposal document are mutable [[namespaceState]]; all undefined names in these namespaces are reserved for future standardization by the W3C.
+        </p>
+      </section>
+      <section id="related-video-object-section">
+        <h3>Related Video Object</h3>
+        <p>
+          A <a>Document Instance</a> MAY be associated with a <a>Related Video Object</a>.
+        </p>
```

Comment:
We should update the "MAY" to "is". In DAPT workflows, as illustrated in the DAPT Requirements diagram, it always start with an Audio/Video Programme.

Reply:
I think I disagree - there are cases where a processor can transform a DAPT document without any specific knowledge of the related video; also we don't require any metadata connecting to a related video. At least in the BBC there may be cases where the same document is associated with _multiple_ videos.

I think we need to look at this whole section via a separate issue though - the whole link to video frames for visual presentation seems irrelevant.

Reply:
I opened https://github.com/w3c/dapt/issues/45